contracts/savings/SavingsManager.sol

```diff
@@ -118,6 +130,34 @@ contract SavingsManager is ISavingsManager, PausableModule {
         IERC20(_mAsset).safeApprove(address(_savingsContract), uint256(-1));
     }
 
+    /**
+     * @dev Freezes streaming of mAssets
+     */
+    function freezeStreams()
+        external
+        onlyGovernor
+        whenStreamsNotFrozen
+    {
+        streamsFrozen = true;
+
+        emit StreamsFrozen();
+    }
+
+    /**
+     * @dev Adds a new revenue recipient
+     * @param _mAsset           Address of underlying mAsset
+     * @param _recipient        Address of the recipient
+     */
+    function addRevenueRecipient(address _mAsset, address _recipient)
```

Comment:
I suppose this could also remove a revenue recipient, right? If so, it might be worth considering to change the function name to `setRevenueRecipient` (similar for the event).

Reply:
Will change to `setRevenueRecipient`